ext/standard/tests/file/tempnam_variation9.phpt

```diff
@@ -0,0 +1,68 @@
+--TEST--
+Test tempnam() function: usage variations - test prefix maximum size
+--SKIPIF--
+<?php
+if(substr(PHP_OS, 0, 3) == "WIN")
+  die("skip Do not run on Windows");
+?>
+--FILE--
+<?php
+/* Testing the maximum prefix size */
+
+echo "*** Testing tempnam() maximum prefix size ***\n";
+$file_path = __DIR__."/tempnamVar9";
+mkdir($file_path);
+
+$pre_prefix = "begin_";
+$post_prefix = "_end";
+$fixed_length = strlen($pre_prefix) + strlen($post_prefix);
+/* An array of prefixes */
+$names_arr = array(
+  $pre_prefix . str_repeat("x", 7) . $post_prefix,
+  $pre_prefix . str_repeat("x", 63 - $fixed_length) . $post_prefix,
+  $pre_prefix . str_repeat("x", 64 - $fixed_length) . $post_prefix,
+  $pre_prefix . str_repeat("x", 65 - $fixed_length) . $post_prefix,
+  $pre_prefix . str_repeat("x", 300) . $post_prefix,
+);
+
+for ($i=0; $i<count($names_arr); $i++) {
+  echo "-- Iteration $i --\n";
+  try {
+    $file_name = tempnam("$file_path", $names_arr[$i]);
+  } catch (Error $e) {
+    echo $e->getMessage(), "\n";
+    continue;
+  }
+
+  $base_name = basename($file_name);
+  echo "File name is => ";
+  print($base_name);
+  echo "\n";
+  echo "File name length is => ";
+  print(strlen($base_name));
+  echo "\n";
+
+  if (file_exists($file_name)) {
+    unlink($file_name);
+  }
+}
+rmdir($file_path);
+
+?>
```

Comment:
The test should have a `--CLEAN--` section removing the folder and all files within it

Reply:
Ack. Added.